heartbeat/pgsql

```diff
@@ -1479,7 +1479,7 @@ set_async_mode_all() {
 }
 
 set_async_mode() {
-    cat $REP_MODE_CONF |  grep -q -e "[,' ]$1[,' ]"
+    cat $REP_MODE_CONF |  grep -q -e "\"$1\""
```

Comment:
echo "'1, 2'" | grep -e "\"$1\"" doesnt return any matches, so it'll probably break in configurations with single-quotes.